synapse/lib/scope.py

```diff
@@ -77,6 +77,15 @@ def get(self, name, defval=None):
 
         return defval
 
+    def __contains__(self, name):
```

Comment:
if we use the above NoValu() paradigm, this becomes superfluous. i'd prefer *not* to have "contains" since almost all use patters will have the same "duplicate hash lookup" issues.